_layouts/team-member.html

```diff
@@ -42,12 +42,46 @@ <h1 class="title">{{ page.firstname }} {{ page.lastname }}
                             {{ link.name }}
                         </a>
                     </li>
-                    {% endfor %}
+                    {% endfor %}    
                 </ul>
             </div>
             {% endif %}
         </div>
     </div>
+
+    <!-- blog posts -->
+    <div class="row">
+        <div class="col">
+            <h2 class="title"> Blog posts that {{ page.firstname }} has written: </h3>
+            <div class="team-links">
+                <ul>
```

Comment:
If you're using `<ul>`, the children should be `<li>` - you can always restyle them to fit your original vibe, but make sure you match things up semantically!